src/delegation/ZoraCreator1155Attribution.sol

```diff
@@ -37,6 +37,8 @@ struct TokenCreationConfig {
     address royaltyRecipient;
     // Fixed price minter address
     address fixedPriceMinter;
+    // Create referral
+    address createReferral;
```

Comment:
the `TOKEN_DOMAIN` and `ATTRIBUTION_DOMAIN` need to have this field added to the corresponding strings